src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -535,11 +670,66 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
                       </Tooltip>
                     </Code>
                     <ExampleBox title="CML Report">
-                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative/cml_cloud_case/pull/11">
+                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative-test/cml-example-cloud/pull/1">
                         <Image src="/img/github/cloud-report.png" alt="GitHub Cloud report example" />
                       </a>
                     </ExampleBox>
-
+        
+                  </Collapser>
+                )}
+                bitbucket={(
+                  <Collapser>
+                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-test/cml-example-cloud">
```

Comment:
- [ ] TODO: @dacbd TPI example

```suggestion
                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-ai/cml-example-cloud-gpu">
```

Reply:
https://github.com/iterative/cml-examples-dev/pull/24